src/profile/main/tag/hotelsCombined.js

```diff
@@ -20,7 +20,7 @@ try{
 
       if(typeof b._corder!="undefined"){
 	//c.push("BookingValue="+b._csubtotal);
-	c.push("BookingValue="+b.checkout.hotel.totalRoomRate);
+	c.push("BookingValue="+b.entity.checkout.hotel.rooms[0].totalPrice.amount);
 	//c.push("Currency="+((b._ccurrency)?b._ccurrency:"USD"));
```

Comment:
@srkurup , please check the legacy implementation for hotelsCombined to confirm whether total booking Cost or Room Rate need to be passed